UPGRADE.md

```diff
@@ -2,6 +2,28 @@
 
 ## dev-develop
 
+### MediaBundle storage configuration
+
+Configuration tree for local storage has changed:
+
+__Before:__
+```
+sulu_media:
+    storage:
+        local:
+            path: '%kernel.project_dir%/var/uploads/media'
+            segments: 10
+```
+
+__After:__
+```
+sulu_media:
+    storages:
```

Comment:
we could avoid the bc break if we use something like:

```
sulu_media:
    storage:
        default: local
        local:
            path: '%kernel.project_dir%/var/uploads/media'
            segments: 10
```

so also all storage configuraiton is available under:

```
bin/console debug:config sulu_media storage
```

Reply:
@chirimoya @danrot what do you think?

Reply:
I think I would make the configuration as close as possible to other bundles. Doctrine can e.g. have multiple entity managers, how is it handled there?

Reply:
the structure of doctrine is similar like the structure in this PR:

![image](https://user-images.githubusercontent.com/1464615/49388733-ea80e800-f724-11e8-9e3d-8d44121ae9b1.png)

and also other bundles of us do it like this